web/pandas/pdeps/0001-purpose-and-guidelines.md

```diff
@@ -71,16 +82,69 @@ Proposing a PDEP is done by creating a PR adding a new file to `web/pdeps/`.
 The file is a markdown file, you can use `web/pdeps/0001.md` as a reference
 for the expected format.
 
-The initial status of a PDEP will be `Status: Under discussion`. This will be changed to
-`Status: Accepted` when the PDEP is ready and has the approval of the core team.
+The initial status of a PDEP will be `Status: Draft`. This will be changed to
+`Status: Under discussion` by the author(s), when they are ready to proceed with the decision
+making process.
 
-#### Accepted PDEP
+#### PDEP Discussion Timeline
+
+A PDEP discussion will remain open for up to 60 days. This period aims to enable participation
+from volunteers, who might not always be available to respond quickly, as well as provide ample
+time to make changes based on suggestions and considerations offered by the participants.
+Similarly, the following voting period will remain open for 15 days.
+
+To enable and encourage discussions on PDEPs, we follow a notification schedule. At each of the
+following steps, the pandas team, and the pandas dev mailing list are notified via GitHub and
+E-mail:
+- Once a PDEP is ready for discussion.
+- After 30 discussion days, with 30 days remaining for discussion.
+- After 45 discussion days, with 15 days remaining for discussion.
+- In case 15 days passed without any new unaddressed comments, the authors may close the discussion period
```

Comment:
```suggestion
- After 45 days, in case 15 days passed without any new unaddressed comments, the authors may close the discussion period
```

Reply:
Since there is a requirement to participate in order to downvote, then we could have a situation where a discussion has not had any objections and proceeding to a vote would be pointless since only those in support could vote.

This is maybe not a bad thing as it puts the onus on members to get involved in the discussion early.

Extending the minimum discussion period is a potential solution to give sufficient opportunity to participate, but it maybe that we should have a notice period for the vote to begin. "voting starts now" is a bit final.

Reply:
to avoid this then maybe we should just remove the requirement to participate in the discussion in order to be able to downvote

Reply:
> Extending the minimum discussion period is a potential solution to give sufficient opportunity to participate, but it maybe that we should have a notice period for the vote to begin. "voting starts now" is a bit final.

It's a good point that when a discussion has been quiet, being able to just "start the voting" with an early vote, sounds a bit abrupt (exactly because the discussion went quiet, people might have forgotten about commenting on the PDEP).

I updated the text around this slightly to just state in general that we expect a notification 15 days before a vote would start. And then this heads up for a nearing vote can either be after 30 days for an early vote (so the end result is the same as Noa's suggestion above that the actual cote can happen after 45 days) or later.

Reply:
Thanks @jorisvandenbossche 

We use the term "unaddressed comment" in the proposal. I changed my review here to "requires changes" recently because of this.

Now, IIRC in the discussion about PDEP-8 (i've not checked the timelines thoroughly and it maybe that I should use a hypothetical case instead) that we had potentially progressed to the vote with unaddressed comments.

Any ideas to cover this scenario or whether we need to at this stage?